src/pages/dao/utils.js

```diff
@@ -0,0 +1,95 @@
+export const TOKEN_DROP_CONTRACT_ADDRESS = 'KT1NhaMvktjR89DcdNw6gYbtW9nXnivn65uq';
+
+export const IPFS_GATEWAY = 'https://ipfs.io/ipfs/';
+
+export const DISTRIBUTION_MAPPING_IPFS_PATH = "QmXDnvkw5ZxNQKKtNaYyve5JuCeM8PfxRM9iAZCi84JKpN";
+
+export const MERKLE_DATA_IPFS_PATHS = {
+    0: "QmR3tiFDi8vgTdvR98jS8jLn2No8EheMG6WTnpbzbupvZm",
+    1: "QmWG2jqiHXFk6Ub7xY7TtjYomxern3SNXuCDVxVeFc88iA",
+    2: "QmZ6P44rhTMxKHqADcW27quQPgvs6h1YTQaz1hS8J3FggF",
+    3: "Qmbm99hPZBQKg5wUhzYTuQUrit5eDgvytqU8J17kpcC71T",
+    4: "QmVBQjr7m25MgF2S7BkmNMtzVNfSuWCaxsYJMXkA7PNhKu",
+    5: "QmTi6E5HJJGZrwKkvBRYMHx2DaVD9mij9rm5cSyDVWv64f",
+    6: "QmbZsvCDiGv4xWeLtMf1wPxjogpvLjaKLwdZe5xQezUWUS",
+    7: "QmbCvvyP2s6nqq1cCW9H617jRGEYoZkV3SoJHCS9sJmsNU",
+    8: "QmRhr7m6zBtU1YFNcLMo34jQVUFdamG3BCGCFjA9AwBEnK"
+};
```

Comment:
I pinned these so that they can be fetched with cache.teia.rocks

Reply:
Those are from the test we did one month ago. We will need to create new ones with the final distribution